pxtlib/emitter/cloud.ts

```diff
@@ -125,6 +125,17 @@ namespace pxt.Cloud {
         })
     }
 
+    export function downloadBuiltSimJsInfoAsync(id: string): Promise<pxtc.BuiltSimJsInfo> {
```

Comment:
async/await?

Reply:
This function modeled after similar ones above, but I'll bring this one into the modern age :)